src/components/ZeroState/index.tsx

```diff
@@ -15,21 +21,4 @@ const ZeroState = ({ Icon, label = '', height = '0px' }) => (
   </article>
 );
 
-ZeroState.propTypes = {
-  /**
-   * Pass in "only" a FDS Icon reference to display it (e.g. Icon={TableIcon})
-   */
-  Icon: PropTypes.func,
-
-  /**
-   * Text inside
-   */
-  label: PropTypes.oneOfType([PropTypes.string, PropTypes.number, PropTypes.node]),
-
-  /**
-   * Min height of section
-   */
```

Comment:
Not sure if you're using `ts-migrate` but it unfortunately doesn't move the prop comments to the type definition.

Reply:
done. Be well!